etc/script/bootstrap.sh

```diff
@@ -58,12 +58,31 @@ gem install bundler
 echo "\n[3] > Updating Bundle (with Fastlane) ...\n"
 BUNDLE_GEMFILE="${PROJECT_DIR}/Gemfile" bundle update
 
+# npm 설치
+if ! command -v npm &> /dev/null; then
+  echo "\n[4] > Installing npm ...\n"
+  brew install node
+fi
+
+# grunt-cli 설치
+cd "${PROJECT_DIR}"
+if ! command -v grunt &> /dev/null; then
+  echo "\n[5] > Installing grunt-cli ...\n"
+  npm install -g grunt-cli
+fi
+
+# 로컬 grunt 설치
+if [[ ! -d node_modules/grunt ]]; then
+  echo "\n[6] > Installing grunt ...\n"
+  npm install
+fi
+
 # mise 설치
-echo "\n[4] > Installing mise ...\n"
+echo "\n[7] > Installing mise ...\n"
 curl https://mise.run | sh
```

Comment:
P4:
```suggestion
if ! command -v mise &> /dev/null; then
  echo "\n[7] > Installing mise ...\n"
  curl https://mise.run | sh
else
  echo "mise is already installed."
fi
```
`make bootostrap` 명령어를 자주 실행하면서 느꼈는데, mise가 이미 설치되었으면 설치 과정을 생략하면 좋겠다는 생각에 추가해보았습니다

Reply:
동의 어 보감~